modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeSelect.ts

```diff
@@ -97,7 +97,7 @@ const generateSelectItems = (_editor: Editor, backstage: UiFactoryBackstage, spe
         return Optional.some<Menu.NestedMenuItemSpec>({
           type: 'nestedmenuitem',
           text: translatedText,
-          disabled: items.length <= 0,
+          enabled: !(items.length <= 0),
```

Comment:
Not sure an array can have a negative length. Shouldn't this be better written as `items.length > 0`